### PR TITLE
[Review] Request from 'digitaltom' @ 'SUSE/connect/review_150922_connect_fixes' 2793

### PR DESCRIPTION
**Pull request by digitaltom**
Please review the following changes:
  * 14cabac use find
  * 2f0594d set release version in rollback
  * ff949e1 try to remove service with obsoleted_service_name also
  * 7023827 fix parameter of synchronize method to be an array of openstructs
  * 375a087 improve docker setup

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@vlewin is your reviewer :dancers: check https://trello.com/c/ZthzXJOb/2793-cnt-fixes-for-migration 
@vlewin: Please review this pull request using our guidelines: 
* test for acceptance criteria / functionality 
* check if the new code is covered with tests 
* check for unintended consequences 
* encourage usage of the boyscout rule 
* make sure the code is architected in the best way 
* check that no unnecessary technical debt got introduced 
* make sure that no unnecessary FIXMEs or TODOs got added

File: lib/suse/connect/zypper.rb [hunk 1]
```diff
@@ -117,6 +117,11 @@ def install_release_package(identifier)
           call("--non-interactive install #{identifier}-release") if identifier
         end
 
+        # rubocop:disable AccessorMethodName
+        def set_release_version(version)
```
Comment (vlewin) [inline lib/suse/connect/zypper.rb]: :) You can also use release_version(version) since it is a valid setter method.

**Comment by vlewin** [comment]
LGTM

**Comment by vlewin** [comment]
@digitaltom Please bump the version and push package to Devel:SCC:suseconnect

**Pull request merged**